pkg/postgres/repository.go

```diff
@@ -3,12 +3,22 @@ package postgres
 import (
 	"database/sql"
 	"fmt"
+	"strings"
 
 	"github.com/smartatransit/scrapedumper/pkg/martaapi"
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 )
 
+type LastestEstimate struct {
+	Direction   martaapi.Direction
+	Line        martaapi.Line
+	TrainID     string
+
+	NextArrival EasternTime
+	EventTime   EasternTime
+}
```

Comment:
The job of the endpoint code will basically be to translate this thing into the types used by the third-rail repo. It'll also need to normalize the station/line/directon names, because scrapedumper doesn't do as much of that as third-rail does today.